src/components/touchables/GenericTouchable.tsx

```diff
@@ -274,11 +272,10 @@ export default class GenericTouchable extends Component<
         shouldActivateOnStart={this.props.shouldActivateOnStart}
         disallowInterruption={this.props.disallowInterruption}
         testID={this.props.testID}
+        {...coreProps}
```

Comment:
Does `AnimatedBaseButton` actually support all the accessibility props from `coreProps`? As far as I know this is the reason for using nested Views inside `*Button` components ([docs](https://docs.swmansion.com/react-native-gesture-handler/docs/api/components/buttons/)).

Reply:
Yea looks like you’re right, I think it would be possible to keep the extra view, but split the style between the 2 views to make sure it lays out as expected if it was only a single view.

For example the flex style needs to be on the outer view while flexDirection should be on the inner one.

Reply:
I thought about doing the same thing when making my own touchable component, but turns out `View` has quite a few style props so I didn't bother in the end ([here's the list](https://gist.github.com/yfunk/f7d8152cb36d6a4b055f16db0ed069b5)).

This would definitely be a nice feature though if you can find a neat way to split the styles.